pubsub/subscriptions/add_users.go

```diff
@@ -26,7 +26,7 @@ import (
 // addUsers adds all IAM users to a subscription.
 func addUsers(projectID, subName string) error {
 	// projectID := "my-project-id"
-	// subName := projectID + "-example-sub"
+	// subName := "my-sub" // subscription id
```

Comment:
Instead of the comment, would you mind renaming the variable `subID`? That way, we don't need the comment at all. Same with `topicName`->`topicID`.

Reply:
That's actually better. Done.